app/outgress/internal/twitch/channel_test.go

```
@@ -0,0 +1,130 @@
+
+func jsonOK(body string) *http.Response {
+	return &http.Response{
+		StatusCode: http.StatusOK,
+		Body:       io.NopCloser(strings.NewReader(body)),
+		Header:     make(http.Header),
+	}
+}
+
+// routeClient fails the test on any request that is not method+path and
+// answers the expected one with body.
+func routeClient(t *testing.T, method, path, body string) *Client {
```

golangci-lint (2.12.2)

[error] func routeClient is unused

(unused)